src/wazuh_db/helpers/wdb_global_helpers.c

```diff
@@ -1048,6 +1048,10 @@ int wdb_remove_agent_db(int id, const char * name) {
     char path[PATH_MAX];
     char path_aux[PATH_MAX];
 
+    if (NULL == name || *name == '\0' || id == OS_INVALID) {
```

Comment:
Is `id == OS_INVALID` validation necessary?

Reply:
Fixed!